crates/libcgroups/src/v2/memory.rs

```diff
@@ -77,6 +77,8 @@ impl Memory {
             stats::parse_single_value(&cgroup_path.join(format!("{}.{}", file_prefix, "current")))?;
         let limit =
             stats::parse_single_value(&cgroup_path.join(format!("{}.{}", file_prefix, "max")))?;
+        let max_usage =
```

Comment:
Please add the error handling if this file doesn't exist, i.e. the users' kernel version doesn't meet it.

Reply:
done, is it ok?

Reply:
Thanks, but won't this hide the actual error?

Reply:
The function `parse_single_value` will return two kinds of error: `WrappedIoError` and `InvalidData`. `WrappedIoError` is what we want to mute. And the file `memory.peak` (also `memory.swap.peak`) is a read-only file provided by kernel, so `InvalidData` error is rare. So here I treat them both as the peak file is missing.

Actually, I found that in cgroups v1, `memsw` part is not using error handling. But mem+swap accounting is not enabled by default (should add `swapaccount=1` to grub), so the situation is similar to here. I'm not quite sure if we need more detailed error handling, but I think it's enough for us.

Please feel free to comment and add suggestions, thanks.

Reply:
Okay, I see, but please cover the test case for the missing file.

Reply:
Others, looks good +1

Reply:
OK, I add a test `test_get_memory_data_with_peak` and kept the origin test for the missing file case. PTAL, thx @utam0k